components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { Button } from '../Button/Button';
+import { cn } from '@/utils/utils';
+import { EntryStatus } from '../EntryStatus/EntryStatus';
+import { ILeagueEntriesProps } from './LeagueEntries.interface';
+import * as React from 'react';
+
+const LeagueEntries = React.forwardRef<HTMLDivElement, ILeagueEntriesProps>(
+  ({ entryNumber, isEliminated = false, isPickSet = false }, ref) => (
+    <div
+      data-testid="LeagueEntryContainerCard"
+      className={cn(
+        'LeagueEntryContainerCard grid h-20 min-w-fit grid-cols-2 justify-between rounded-lg border bg-card p-4 text-card-foreground shadow-sm outline dark:border-zinc-800',
```

Comment:
CSS classes should be lowercased and hyphen separated. Update all of them in this file. `league-entries`

Reply:
Updated in latest commit.